<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmeticPage</class>
 <widget class="QWidget" name="ArithmeticPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>812</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
      <widget class="QLabel" name="Arithmetic_Title">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Arithmetic Operations</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Power_Title">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Power Operations</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputsLayout">
     <item>
      <widget class="QGroupBox" name="Numbers_Group">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Numbers (one per line)</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPlainTextEdit" name="Numbers_TextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Example:
5
3
2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="powerWrapper">
       <item>
        <widget class="QGroupBox" name="Power_Group">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Power</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QFormLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="Base_Label">
              <property name="text">
               <string>Base</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="Base_LineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="Exponent_Label">
              <property name="text">
               <string>Exponent</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="Exponent_LineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="Power_Button">
            <property name="text">
             <string>Calculate Power</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="operationsLayout">
     <item>
      <widget class="QPushButton" name="Add_Button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Subtract_Button">
       <property name="text">
        <string>Subtract</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Multiply_Button">
       <property name="text">
        <string>Multiply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Divide_Button">
       <property name="text">
        <string>Divide</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="Result_Group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="Result_Value">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>â€”</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
